Javascript/README.md

```diff
@@ -49,6 +49,23 @@ if (time < 60) {
 }
 ```
 
+There is an objective reason for this. If you write the following code
+
+```javascript
+function myFunction()
+{
+	return
+	{
+		property1: 'value1',
+		property2: 'value2'
+	};
+}
+```
+
+`return\n` will be interpreted as `return;\n` and your function will be broken. Thus, for consistency matter, we want
+any curly braces to be on the same line as whatever they open.
```

Comment:
Well, to me, this is a sloppy example. I don't see how it can illustrate or justify the **Braces** rule. The developer here clearly wanted the braces to be an object definition.